com.jagex.RuneScape.json

```diff
@@ -0,0 +1,148 @@
+        },
+        {
+            /* Required to build glew 1.10 */
+            "name": "libxmu",
+            "cleanup": ["/include", "/lib/debug", "/lib/pkgconfig", "/share"],
+            "sources": [
+                {
+                    /* Use latest version */
+                    "type": "archive",
+                    "url": "https://xorg.freedesktop.org/releases/individual/lib/libXmu-1.1.2.tar.bz2",
+                    "sha256": "756edc7c383254eef8b4e1b733c3bf1dc061b523c9f9833ac7058378b8349d0b"
+                }
+            ]
+        },
+        {
+            /* No textures will be displayed without this */
+            "name": "s2tc",
+            "config-opts": ["--disable-tools"],
+            "cleanup": ["/include", "/lib/debug", "/lib/pkgconfig"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/divVerent/s2tc.git",
+                    "commit": "f6ec862d7594e29ae80a6e49f66ad4c76cf09abc"
+                }
+            ]
+        },
+        {
+            "name": "glew",
+            "no-autogen": true,
+            "make-args": [
+                "GLEW_PREFIX=/app",
+                "GLEW_DEST=/app",
+                "LIBDIR=/app/lib"
+            ],
+            "make-install-args": [
+                "GLEW_PREFIX=/app",
+                "GLEW_DEST=/app",
+                "LIBDIR=/app/lib"
+            ],
+            "cleanup": ["/include", "/lib/debug", "/lib/pkgconfig"],
+            "sources": [
+                {
+                    /* Must be version 1.10 */
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/glew/glew/1.10.0/glew-1.10.0.tgz",
+                    "sha256": "99c41320b63f6860869b5fb9af9a1854b15582796c64ee3dfd7096dc0c89f307"
+                },
+                {
+                    "type": "patch",
+                    "path": "glew-no-static-lib.patch"
+                }
+            ]
+        },
+        {
+            "name": "runescape",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install apply_extra /app/bin",
+                "install runescape.sh /app/bin/runescape",
+                "cp /usr/bin/ar /app/bin",
+                "cp /usr/lib/libbfd-*.so /app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "ar x runescape.deb",
+                        "tar xf data.tar.xz",
+                        "rm -f runescape.deb control.tar.gz data.tar.xz debian-binary _gpgbuilder",
+                        "mv usr/share/games/runescape-launcher/runescape .",
+
+                        "mkdir -p export/share/applications",
+                        "sed -i 's|/usr/bin/runescape-launcher %u|runescape|g' usr/share/applications/runescape-launcher.desktop",
```

Comment:
This is removing `%u`, what are urls used for, probably want to keep that support?